i18n/en.pot

```diff
@@ -192,8 +192,8 @@ msgstr "Showing all values for this dimension"
 msgid "Program stage: {{stageName}}"
 msgstr "Program stage: {{stageName}}"
 
-msgid "No dimensions found for '{{searchTerm}}'"
-msgstr "No dimensions found for '{{searchTerm}}'"
+msgid "No dimensions found for '{{- searchTerm}}'"
```

Comment:
What does the "-" do?

Reply:
Unescapes the string to prevent `/` etc in the searchTerm to be shown as unicode chars. https://www.i18next.com/translation-function/interpolation#unescape

Reply:
Could you add that to the PR description (and final merge commit)